src/components/content/heroes/aurora/weather-easter-egg/sun.tsx

```diff
@@ -0,0 +1,129 @@
+import SUN from '../../../../../assets/images/aurora/sun.png';
+import REFLECTION from '../../../../../assets/images/aurora/sun-reflection.png';
+import SUNSHINE from '../../../../../assets/images/aurora/sun-shine.png';
+import styled, { keyframes } from 'styled-components';
+import { useEffect, useState } from 'react';
+import React from 'react';
+import { getSunTime } from './weather-api';
+import { getSunlightPercentage } from './sun-percentage-calculator';
+import { Flare, FlareType } from '../flare';
+import { DefaultFlares } from '../default-flares';
+
+const rotatingAnimation = keyframes`
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+`;
+
+const AuroraSunShineDiv = styled.div`
+  background-image: url(${SUNSHINE});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  animation: ${rotatingAnimation} 60s linear infinite;
+  width: 200%;
+  height: 200%;
+  left: -50%;
+  top: -50%;
+  position: absolute;
+`;
+
+const AuroraSunDiv = styled.div<{ timePercent: number }>`
```

Comment:
What to do you think about adding a transition here, to let the sun fly into its correct position instead of jumping there (once the api request has responded)?